.github/workflows/unit.yaml

```diff
@@ -17,6 +17,21 @@ jobs:
       with:
         go-version: ~1.20.0
 
+    - name: Create kubebuilder directory
```

Comment:
Why is this all being installed to a kubebuilder directory?

If it's a requirement we should have a comment explaining why because App Routing doesn't actually use the kubebuilder framework.

Reply:
It seems fine to keep as is just documenting the reason with a comment would be good.

Reply:
Added comment for GH action